.github/workflows/syncSnapDocs.yml

```diff
@@ -0,0 +1,17 @@
+name: Sync docs with gitbook
+
+on:
+  push:
+    branches: ['github-actions-readme']
```

Comment:
I guess after testing is done, this is supposed to be replaced with `main` branch, right?

Reply:
yes